docs/book.toml

```diff
@@ -1,13 +1,6 @@
 [book]
-authors = ["mattwparas"]
+authors = ["mattwparas", "fominok"]
 language = "en"
 multilingual = false
 src = "src"
-title = "Steel"
-
-# https://github.com/slowsage/mdbook-pagetoc
-[preprocessor.pagetoc]
-
-[output.html]
-additional-css = ["theme/pagetoc.css"]
-additional-js  = ["theme/pagetoc.js"]
```

Comment:
Could we keep this in?

Reply:
Ah, I didn't figure out what was that artifact and removed it

it was just an empty TOC, I addressed that in .css